index.html

```diff
@@ -1,31 +1,65 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="./src/css/main.css">
     <title>Axit</title>
 </head>
+
 <body>
     <header class="c-header">
         <div class="l-container">
-       <div class="c-header__wrapper">
-        <div class="c-header__logo">
-           <a href="#" class="c-header__logo-link">
-            <img class="c-header__logo-image" src="./src/assets/images/logo.png" alt="logo">
-           </a>
+            <div class="c-header__wrapper">
+                <div class="c-header__logo">
+                    <a href="#" class="c-header__logo-link">
+                        <img class="c-header__logo-image" src="./src/assets/images/logo.png" alt="logo">
+                    </a>
+                </div>
+                <nav class="c-navigation">
+                    <ul class="c-navigation__list">
+                        <li class="c-navigation__item"><a href="#" class="c-navigation__link">Features</a></li>
+                        <li class="c-navigation__item"><a href="#" class="c-navigation__link">About</a></li>
+                        <li class="c-navigation__item"><a href="#" class="c-navigation__link">Pricing</a></li>
+                        <li class="c-navigation__item"><a href="#" class="c-navigation__link">Reviews</a></li>
+                        <li class="c-navigation__item"><a href="#" class="c-navigation__link">Contact</a></li>
+                    </ul>
+                </nav>
+            </div>
         </div>
-        <nav class="c-navigation">
-            <ul class="c-navigation__list">
-                <li class="c-navigation__item"><a href="#" class="c-navigation__link">Features</a></li>
-                <li class="c-navigation__item"><a href="#" class="c-navigation__link">About</a></li>
-                <li class="c-navigation__item"><a href="#" class="c-navigation__link">Pricing</a></li>
-                <li class="c-navigation__item"><a href="#" class="c-navigation__link">Reviews</a></li>
-                <li class="c-navigation__item"><a href="#" class="c-navigation__link">Contact</a></li>
-            </ul>
-        </nav>
-       </div>
-    </div>
     </header>
+    <main>
+        <section class="c-hero-banner">
+            <div class="l-container">
+                <div class="c-hero-banner__wrapper">
+                    <div class="c-hero-banner-content">
+                        <div class="c-hero-banner-content__logo">
+                            <img class="c-hero-banner-content__logo-img" src="./src/assets/images/logo.png" alt="logo">
+                        </div>

```

Comment:
Why logo is not separate component? Why logo must be part of header? This way we need to use logo in header.